Sources/Atomic.swift

```diff
@@ -107,80 +107,114 @@ internal struct UnsafeAtomicState<State: RawRepresentable> where State.RawValue
 /// `Lock` exposes `os_unfair_lock` on supported platforms, with pthread mutex as the
 // fallback.
 internal class Lock {
+	// Both `UnfairLock` and `PthreadLock` use `ManagedBufferPointer` to allocate inline
```

Comment:
Why use `ManagedBufferPointer` instead of an `UnsafeMutablePointer`? This seems more complex.